samples/charts/financial-chart/multiple-data/Services/StockHistory.cs

```diff
@@ -0,0 +1,112 @@
+                await getTeslaStock()
+            };
+
+            return dataSources;
+        }
+
+        /** gets Amazon stock OHLC prices from a .JSON file */
+        public static async Task<StockItem[]> getAmazonStock() {
+            var url = "https://static.infragistics.com/xplatform/data/stocks/stockAmazon.json";
+
+            //TODO
+            // setting data intent for Series Title, e.g. FinancialChart usage
+            //(stockData as any).__dataIntents = {
+            //  close: ["SeriesTitle/Amazon"]
+            //};
+            // console.log("fetchAmazonStock: ", stockData.length);
+
+            var http = new HttpClient();
+            var data = await http.GetFromJsonAsync<StockItem[]>(url);
+            //Console.WriteLine("Label" + " " + data.Length);
+            //Console.WriteLine("Date" + " " + data[0].Date);
+            return data;
+                   
+        }
+
+        /** gets Tesla stock OHLC prices from a .JSON file */
+        public static async Task<StockItem[]> getTeslaStock()
+        {
+            var url = "https://static.infragistics.com/xplatform/data/stocks/stockTesla.json";
+
+            //TODO
+            // setting data intent for Series Title, e.g. FinancialChart usage
+            //(stockData as any).__dataIntents = {
+            //  close: ["SeriesTitle/Amazon"]
+            //};
+            // console.log("fetchAmazonStock: ", stockData.length);
+
+            var http = new HttpClient();
+            var data = await http.GetFromJsonAsync<StockItem[]>(url);
+
+            return data;
+
+        }
+
+        /** gets Microsoft stock OHLC prices from a .JSON file */
+        public static async Task<StockItem[]> getMicrosoftStock()
+        {
+            var url = "https://static.infragistics.com/xplatform/data/stocks/stockMicrosoft.json";
+
+            //TODO
+            // setting data intent for Series Title, e.g. FinancialChart usage
+            //(stockData as any).__dataIntents = {
+            //  close: ["SeriesTitle/Amazon"]
+            //};
+            // console.log("fetchAmazonStock: ", stockData.length);
+
+            var http = new HttpClient();
+            var data = await http.GetFromJsonAsync<StockItem[]>(url);
+
+            return data;
+
+        }
+
+        public static async Task<StockItem[]> getGoogleStock()
+        {
+            var url = "https://static.infragistics.com/xplatform/data/stocks/stockGoogle.json";
+
+            //TODO
+            // setting data intent for Series Title, e.g. FinancialChart usage
+            //(stockData as any).__dataIntents = {
+            //  close: ["SeriesTitle/Amazon"]
+            //};
+            // console.log("fetchAmazonStock: ", stockData.length);
+
+            var http = new HttpClient();
+            var data = await http.GetFromJsonAsync<StockItem[]>(url);
+
+            return data;
+
+        }
+    }
+
+    public class StockItem
+    {
+        public string Date { get; set; }
```

Comment:
did you get the Date to show up on XAxis? If  not then add TODO comment with this issue